pages/previewModel.js

```diff
@@ -120,13 +124,14 @@ function updatePreview() {
     // give them all the same special ID so that we can easily remove them later.
     // We then do something similar for link (CSS) elements.
     clearRemovableHeadElements();
-    var scriptElements = content.getElementsByTagName('script')
-    for (var i = 0; i < scriptElements.length; i++) {
+    var i;
+    var scriptElements = content.getElementsByTagName('script');
+    for (i = 0; i < scriptElements.length; i++) {
         addHeadScript(scriptElements[i].src, scriptElements[i].innerHTML);
     }
 
-    var linkElements = content.getElementsByTagName('link')
-    for (var i = 0; i < linkElements.length; i++) {
+    var linkElements = content.getElementsByTagName('link');
```

Comment:
I was also writing PowerShell code that day, and PowerShell doesn't need semi-colons. Sorry about that.